proto/dropbox/dropbox.proto

```diff
@@ -0,0 +1,79 @@
+syntax = "proto3";
+
+package dropbox;
+
+service Dropbox {
+  rpc GetApproverInfo(GetApproverInfoRequest) returns(GetApproverInfoResponse) {
```

Comment:
why not just send the `ApproverInfo` message as the response?

Reply:
Don't know, changed that.

Reply:
Not yet. You don't have to though, it just seems a bit redundant.